packages/headless/src/features/analytics/search-action-cause.ts

```diff
@@ -200,18 +200,6 @@ export enum SearchPageEvents {
    * Identifies the custom event that gets logged when a snippet suggestion for a related question is collapsed.
    */
   collapseSmartSnippetSuggestion = 'collapseSmartSnippetSuggestion',
-  /**
-   * Identifies the custom event that gets logged when the user presses "show more" on a snippet suggestion for a related question.
-   *
-   * @deprecated
-   */
-  showMoreSmartSnippetSuggestion = 'showMoreSmartSnippetSuggestion',
```

Comment:
I'm of the opinion that we should only modify the stuff we make a PR for, but thanks for removing those anyway :D